Tests/CodTests/CodBasicTests.swift

```diff
@@ -161,3 +198,9 @@ extension Array {
 		}
 	}
 }
+
+extension CodingUserInfoKey {
+	static var randomValue: CodingUserInfoKey {
+		return CodingUserInfoKey(rawValue: "RandomValue")!
+	}
+}
```

Comment:
Can you move this into the test itself? I don't really see it being useful outside of it and the ergonomics of going `decoder.userInfo[randomValue]` seem fine to me, with the code to set it up also being simpler:

```swift
let randomValue = CodingUserInfoKey(rawValue: "RandomValue")!
```

Ideally, I would also prefer names that are a little more appropriate, like "testUserInfoKey" and "UserInfoTestCodable" or something like that.